src/@chakra-ui/gatsby-plugin/theme.js

```diff
@@ -52,7 +46,7 @@ const {
 
 const config = {
   initialColorMode: 'light',
```

Comment:
```suggestion
  initialColorMode: 'system',
```
IIRC from the simulator